packages/react/src/components/ThemeProvider/__tests__/ThemeProvider.test.tsx

```diff
@@ -21,22 +21,16 @@ describe('ThemeProvider', () => {
   });
 
   it('wraps the App in [data-amplify-theme="default-theme"]', () => {
-    jest.spyOn(global.document.documentElement, 'setAttribute');
     const { container } = render(
       <ThemeProvider>
         <App />
       </ThemeProvider>
     );
 
-    expect(global.document.documentElement.setAttribute).toHaveBeenCalledWith(
+    expect(container.querySelector(`[data-amplify-theme]`)).toHaveAttribute(
```

Comment:
```suggestion
    expect(global.document.documentElement).toHaveAttribute(
```
No need to query? The root element is kinda special